src/main/java/com/graphhopper/converter/api/NominatimEntry.java

```diff
@@ -141,12 +160,22 @@ public Address() {
         public String village;
         @JsonProperty
         public String hamlet;
-        @JsonProperty("road")
-        public String road;
-        @JsonProperty("pedestrian")
-        public String pedestrian;
         @JsonProperty("house_number")
         public String houseNumber;
+        @JsonProperty
+        public String postcode;
+
+        // Possible Street names TODO: Not sure what tags can be returned here
+        @JsonProperty
+        public String road;
+        @JsonProperty
+        public String pedestrian;
+        @JsonProperty
+        public String path;
+        @JsonProperty
+        public String footway;
+        @JsonProperty
+        public String construction;
```

Comment:
Oh, this is ugly ... and I also did not find documentation for this. One further tag you can add is cycleway: http://nominatim.openstreetmap.org/search.php?q=bayreuth%20fahrradhighway&polygon=1&format=json&addressdetails=1